linkatos/firebase.py

```diff
@@ -2,6 +2,9 @@
 
 
 def initialise(api_key, project_name):
+    """
+    Connects to the Firebase project.
```

Comment:
I used project because I think it's what they called them on Firebase, but I might be wrong I'll check